packages/readline.rb

```diff
@@ -3,16 +3,16 @@
 class Readline < Package
   description 'The GNU Readline library provides a set of functions for use by applications that allow users to edit command lines as they are typed in.'
   homepage 'http://cnswww.cns.cwru.edu/php/chet/readline/rltop.html'
-  version '8.1'
+  version '8.1-1'
```

Comment:
Maybe split these patches into separate PRs?

Reply:
Can do

Reply:
Hmmm... actually... these all build on each-other. SWI-prolog depends on readline being linked to libtinfo and on pcre being built with --enable-unicode-properties. readline depends on ncurses which is updated too. My gut says it's better to keep the change 'atomic', and I'm unsure how to go about splitting these up nicely. I'm open to ideas!

Reply:
I was able to eliminate the change to ncurses, but the the change to pcre is necessary to pass the tests and linking readline to libtinfo is needed to prevent a build failure without resorting to patching the files generated by CMake or resorting to other (IMO uglier) hacks. Let me know if you have any ideas for how to split this up further or if I can mark this resolved. :-)

Reply:
Binaries added for all of these.